src/TextBox/BaseBox.ts

```diff
@@ -15,27 +17,131 @@ class BaseBox extends Component <BoxConfig> {
         super(config);
     }
 
-    render() {
+    render(): void {
+        if (this.exists) {
+            return;
+        }
         console.log('TextBox render: ' + this.config.id);
+        this.exists = true;
         document.body.appendChild(this.element);
+        this.show();
```

Comment:
If

```javascript
let tb = new TextBox();
tb.render();
tb.hide();
tb.render();
```

, the textbox can't show.

Reply:
done.
https://github.com/cloxnu/textbox/pull/12/commits/dd4355802883cfc02cab53449d5af8232861709b#diff-55b391c93a4d6bbb6929c1b3bc2ab3556c2ebc65eb3eae9c0bb536614e56d5e8R22